docs/package_list.md

```diff
@@ -0,0 +1,41 @@
+| Package name                                                 | Package hash                                                 |
+| ------------------------------------------------------------ | ------------------------------------------------------------ |
+| protocol/fetchai/gym/1.0.0                                   | bafybeihpjm2sgnrpuwaqicikw4aybltm7xrjmf7cscpp2cy2xdoi6pekbq  |
```

Comment:
I think that (at least) the hashes should be displayed in fixed-width font.

Reply:
Sorted